segway/_util.py

```diff
@@ -374,19 +387,23 @@ def find_segment_starts(data, output_label):
     else:
         len_data = len(data)
 
-    # unpack tuple, ignore rest
-    seg_diffs = absolute(diff(data))
-
     # if output_label is set to "full" or "subseg"
     if output_label != "seg":
+        # Equivalent to diff(data) != 0, so True where labels change
+        # Applied to both rows (seg and subseg) separately
+        seg_diffs = (data[:, 1:] != data[:, :-1])
```

Comment:
This line is duplicated in both conditions. I would move it out and use the `if output_label != "seg"` conditional to apply the last change (the `maximum` call).

Reply:
The first case has the expression `data[:, 1:] != data[:, :-1]` while the second case has the expression `data[1:] != data[:-1]` due to different dimensions of `data`. Is there a single expression that can apply to both cases? 
Reshaping in the first case is an option then applying the latter expression is an option, but it requires reshaping back so I don't think it's significantly simplifying.

Reply:
My mistake. Keep as is. This is fine.